### DISCUSSION TITLE
Working with split screen and tabs

### DISCUSSION THREAD

**Discussion by daveodonnell** [microsoft/MicrosoftEdge-Extensions#102] [Q&A]
The latest version of edge includes a split screen feature. Are there any APIs that an extension can use to know if the split screen is opened and/or closed.
Calling chrome.tabs.query({}, callback) does not include the split screen tab
Is there any way for a background service worker to find the split screen tab?

**Comment by Bhuvana-Priya** [comment]
Hi @daveodonnell,

We are looking into this and will get back to you with any further updates.

Thank you

**Reply by daveodonnell** [reply]
Hi @Bhuvana-Priya 
Are there any updates?

**Reply by mihohoi0322** [reply]
Hi, @Bhuvana-Priya 
Are there any updates regarding this issue? Also, are there any additional plans for the future roadmap?

**Reply by so-nkbys** [reply]
Hello, @Bhuvana-Priya -san
If you have any new information, please share it with us.
We need your help.

**Reply by xylin-dev** [reply]
+1 any news? I really need this feature.

**Comment by Andrew-J-Larson** [comment]
A thought, can anyone test? https://github.com/uBlockOrigin/uBlock-issues/issues/2945#issuecomment-1936749827

> I think I figured out a work around.
> 
> Using the active tab and window, you should check the results of `document.hasFocus()`.
> 
> If true, then you're using the main window (or left split tab), but if false, you'll want to check each "inactive" tabs in the current window where `document.hasFocus()` ends up as true, being the second tab (main window's right split tab).
> 
> Then, you should be able to find the split tab, right?
